### DISCUSSION TITLE
Skip migration by hash

### DISCUSSION THREAD

**Discussion by Niccolum** [sqlalchemy/alembic#1164] [Usage Questions]
Hello,

Thanks for that thing like alembic and sqlalchemy in general.

My question is - is it possible to apply migration, but skip it? It is something like django migration with --zero flag.

Why i need this. I have some migrations, which fill some tables initial data. Some of them use remote sources and it can works about few minutes.

I want to test my migrations, but want to skip some hashes of my existing migrations. Sadly, but i can't find something, which can help me.

Also i ask about in [pytest-alembic](https://github.com/schireson/pytest-alembic/issues/71), but they also don't know that.

**Comment by CaselIT** [comment]
Hi,

I don't think there is anything automatic here. But you can probably use stamp to change what migration is saved in the alembic version table
https://alembic.sqlalchemy.org/en/latest/api/commands.html#alembic.command.stamp
